bootstrap.sh

```diff
@@ -101,103 +101,7 @@ echo ""\""google-cloud-gapic-bom"\"": "\""0.0.0"\""," >> .release-please-manifes
-      #updating gapic bom pom.xml with the snapshot version
-      version_line="${version_line/${module_snapshot_version}/${snapshot_version}}"
-    fi
-
-  if ! [[ "${module_snapshot_version}" =~ .*SNAPSHOT ]]; then
-    # increment the third digit of the version and overwrite it.
-    snapshot_version=$(echo ${module_released_version} |  awk -F'.' '{print $1"."$2"."$3+1}' |  sed s/[.]$//)
-
-    mvn -B -ntp -f ${pom_directory} -U versions:set -DnewVersion=${snapshot_version}
-    mvn -B -ntp -f ${bom_directory} -U versions:set -DnewVersion=${snapshot_version}
-    #updating gapic bom pom.xml with the snapshot version
-    version_line="${version_line/${module_snapshot_version}/${snapshot_version}}"
-  fi
-
-  #for modules which have artifacts with different versions, we need to bump to snapshot version
-  cat ${version_file} | while read line
-    do
-       if [[ ${line} =~ [0-9] ]] && ! [[ ${line} == *"SNAPSHOT"* ]] ; then
-         artifact_name=$(echo "${line}" | awk -F':' '{print $1}')
-         old_version=$(echo "${line}" | awk -F':' '{print $3}')
-         new_version=$(echo ${old_version} |  awk -F'.' '{print $1"."$2"."$3+1}' |  sed s/[.]$//)
-         new_version="${new_version}-SNAPSHOT"
-         sed -i.bak "s|${artifact_name}:${old_version}:${old_version}|${artifact_name}:${old_version}:${new_version}|" ${version_file}
-         artifact_directory="${bom_directory}/../${artifact_name}"
-         mvn -B -ntp -f ${artifact_directory} -U versions:set -DnewVersion=${new_version}
-       fi
-    done
-
-  cat ${version_file} | while read line
-      do
-         if [[ ${line} =~ [0-9] ]] && [[ ${line} == *"SNAPSHOT"* ]] ; then
-           artifact_name=$(echo "${line}" | awk -F':' '{print $1}')
-           old_version=$(echo "${line}" | awk -F':' '{print $2}')
-           new_version=$(echo ${old_version} |  awk -F'.' '{print $1"."$2"."$3+1}' |  sed s/[.]$//)
-           artifact_directory="${bom_directory}/../${artifact_name}"
-           mvn -B -ntp -f ${artifact_directory} -U versions:set -DnewVersion=${new_version}
-         fi
-      done
-
-  #concatenating module name and module version
-  rp_manifest_line=""\""${module}"\"": "\""${module_released_version}"\"""
-
-  rp_config_line+=""\""${module}"\"": {\n\
-        "\""component"\"": ""\""${artifactName_config}"\"","\n\
-        "\""skip-github-release"\"": "true"\n\
-       }"
-
-  #adding " , " where it's necessary
-  if [[ $num_modules -gt 1 ]]; then
-    rp_manifest_line+=","
-    rp_config_line+=",\n    "
-    num_modules=$((num_modules-1))
-  fi
-
-  #adding the line to manifest config file
-  echo "${rp_manifest_line}" >> .release-please-manifest.json
-done
-
-echo "}" >> .release-please-manifest.json
-
-awk -v "packagesList=$rp_config_line" '{gsub(/ALL_PACKAGES/,packagesList)}1' \
-    ../../release_please_config_raw.json > release-please-config.json
+bash generation/release_please_config_generation.sh
```

Comment:
Can you rename the script as "generate_release_please_config.sh"?

I put the verb first in other scripts.

Reply:
Done.